README.md

```diff
@@ -549,3 +572,37 @@ Extensions,
 Data,
   {  "instances": [    [6.8,  2.8,  4.8,  1.4],    [6.0,  3.4,  4.5,  1.6]  ]}
 ```
+
+PRs:
+- https://github.com/kserve/kserve/pull/2792
+
+### Drift detection
```

Comment:
Is the `Drift Detector Service` their own CRD? Or some "KSVC"?

Reply:
Some KSVC